cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -154,10 +151,9 @@ public static void CopyTo(this IBuffer source, uint sourceIndex, byte[] destinat
             if (destination == null) throw new ArgumentNullException(nameof(destination));
             if (count < 0) throw new ArgumentOutOfRangeException(nameof(count));
             if (destinationIndex < 0) throw new ArgumentOutOfRangeException(nameof(destinationIndex));
-            if (source.Length == 0 && count == 0) return;
-            if (source.Capacity <= sourceIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity);
+            if (source.Capacity < sourceIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity);
```

Comment:
these should be source.Length as well

Reply:
Okay I was basing off of documentation/pre-existing, but the only argument I see for capacity is if you know the buffer is partially full, you could make a blank array from the unused part (a span starting at an index greater than length but less than capacity).

Reply:
Buffers which require setting the "length of valid data" explicitly (e.g. native `IBuffer` implementations) may not have their `Length` up to date during all times when this API is called. Changing it to be more restrictive and not allow access to the full buffer would be a breaking change I don't think you should take.